### PR TITLE
named arguments -> named parameters

### PR DESCRIPTION
**Pull request by bergus**
inspired by http://stackoverflow.com/questions/38072740/es6-arrow-function-no-duplicate-named-arguments

### CODE REVIEW

File: manuscript/03-Functions.md [hunk 1]
```diff
@@ -786,14 +786,14 @@ One of the most interesting new parts of ECMAScript 6 is the *arrow function*. A
...
+* **No duplicate named parameters** - arrow functions cannot have duplicate named parameters in strict or nonstrict mode, as opposed to nonarrow functions that cannot have duplicate named parameters only in strict mode.
 
 There are a few reasons for these differences. First and foremost, `this` binding is a common source of error in JavaScript. It's very easy to lose track of the `this` value inside a function, which can result in unintended program behavior, and arrow functions eliminate this confusion. Second, by limiting arrow functions to simply executing code with a single `this` value, JavaScript engines can more easily optimize these operations, unlike regular functions, which might be used as a constructor or otherwise modified.
 
 The rest of the differences are also focused on reducing errors and ambiguities inside of arrow functions. By doing so, JavaScript engines are better able to optimize arrow function execution.
 
-I> Note: Arrow functions also have a `name` property that follows the same rule as other functions.
+I> Note: Arrow functions can also have a `name` property that follows the same rule as other functions.
```
Comment (nzakas) [inline manuscript/03-Functions.md]: This `can` is incorrect. Arrow functions always have a `name` property.

Comment (bergus) [inline manuscript/03-Functions.md]: Aw shucks, I really should've split this up in separate commits - it was just a quick fix.

However I'm pretty sure that arrow functions don't have `name` properties by default, can you point me to the spec where it is created? They are truly anonymous by default, they only get the property when their literal happens to be part of an assignment (etc).

Comment (nzakas) [inline manuscript/03-Functions.md]: All functions have `name` properties. Try it yourself:

```js
let arrow = () => {};
console.log(arrow.name);
```

Comment (bergus) [inline manuscript/03-Functions.md]: That's what I said, it gets a name when it's part of an assignment. Try something else:
```js
(arrow => console.log("name" in arrow, arrow.name))(()=>{});
```

Comment (nzakas) [inline manuscript/03-Functions.md]: Your understanding isn't quite correct. There is a `name` property that all functions inherit from `Function.prototype`. In an assignment context, a function gets an own `name` property and otherwise uses the prototype `name` property. In either case, the function has a `name` property. See https://tc39.github.io/ecma262/#sec-function-instances-name

> Anonymous functions objects that do not have a contextual name associated with them by this specification do not have a name own property but inherit the name property of %FunctionPrototype%.

Can you please remove "can" so I can merge this now?

Comment (bergus) [inline manuscript/03-Functions.md]: OK, "have a property" sounded like "own property" to me. If you meant that they at least inherit `Function.prototype.name`, it's OK.

**Comment by bergus** [comment]
> Can you please remove "can" so I can merge this now?

- [x] done
